src/meta/src/rpc/service/notification_service.rs

```diff
@@ -81,8 +81,10 @@ where
                     .await
                     .map_err(|e| e.to_grpc_status())?;
 
-                let cluster_guard = self.cluster_manager.get_cluster_core_guard().await;
-                let nodes = cluster_guard.list_worker_node(WorkerType::ComputeNode, Some(Running));
+                let nodes = self
+                    .cluster_manager
+                    .list_worker_node(WorkerType::ComputeNode, Some(Running))
+                    .await;
```

Comment:
If a compute node starts between line 84 and line 98, frontend who calls `subscribe` rpc will not get this compute node.

Reply:
I got it, I will add a comment on it.